server/commands.lua

```diff
@@ -28,6 +28,41 @@ lib.addCommand('blips', {
     TriggerClientEvent('qbx_admin:client:blips', source)
 end)
 
+lib.addCommand('admincar', {
+    help = 'Buy Vehicle',
+    restricted = config.saveVeh,
+}, function(source)
+    local src = source
+    local player = exports.qbx_core:GetPlayer(src)
+    local vehicles = exports.qbx_core:GetVehiclesByName()
+    local vehModel, vehicle, props = lib.callback.await('qbx_admin:client:GetVehicleInfo', src)
+
+    if vehicle == 0 then
```

Comment:
Change this to if not vehicle (cache.vehicle returns either number or false)

Reply:
```suggestion
    if not cache.vehicle then
```